cli_client/python/timesketch_cli_client/commands/sketch.py

```diff
@@ -53,3 +55,42 @@ def create_sketch(ctx, name, description):
         description = name
     sketch = api_client.create_sketch(name=name, description=description)
     click.echo(f"Sketch created: {sketch.name}")
+
+
+@sketch_group.command("add_event")
+@click.option("--message", required=True, help="Message of the event.")
+@click.option(
+    "--date",
+    required=True,
+    help="Date of the event (ISO 8601). Example: 2023-03-08T10:59:24+00:00",
+)
+@click.option(
+    "--attributes",
+    required=False,
+    help="Attributes of the event. Example: key1=value1,key2=value2",
+)
+@click.option(
+    "--timestamp_desc",
+    required=False,
+    help="Timestamp description of the event.",
+)
+@click.pass_context
+def add_event(ctx, message, date, attributes, timestamp_desc="Timecketh-cli"):
+    """Add an event to the sketch."""
+    sketch = ctx.obj.sketch
+
+    if attributes:
+        attributes = attributes.split(",")
+        attributes = [attribute.split("=") for attribute in attributes]
+        attributes = {
+            key: value for key, value in attributes
+        }  # pylint: disable=unnecessary-comprehension
+    if not attributes:
+        attributes = {}
+    sketch.add_event(
+        message=message,
+        date=date,
+        timestamp_desc=timestamp_desc,
+        attributes=attributes,
+    )
+    click.echo(f"Event added to sketch: {sketch.name}")
```

Comment:
Also output the event ID?

Reply:
not possible, at the moment we only return the following JSON:
````
{'meta': {}, 'objects': [{'color': 'F37991', 'created_at': '2023-03-08T12:46:24.472587', 'datasources': [], 'deleted': None, 'description': 'internal timeline for user-created events', 'id': 19, 'label_string': '', 'name': 'Manual events', 'searchindex': {'created_at': '2023-03-08T12:46:24.047640', 'deleted': None, 'description': 'internal timeline for user-created events', 'id': 9, 'index_name': '49a318b0ba17867fd71b50903774a0c8', 'label_string': '', 'name': 'Manual events', 'status': [{'created_at': '2023-03-08T14:35:27.328896', 'id': 81, 'status': 'ready', 'updated_at': '2023-03-08T14:35:27.328896'}], 'updated_at': '2023-03-08T12:46:24.047640', 'user': {'active': True, 'admin': True, 'groups': [], 'username': 'dev'}}, 'status': [{'created_at': '2023-03-08T14:35:27.362493', 'id': 73, 'status': 'ready', 'updated_at': '2023-03-08T14:35:27.362493'}], 'updated_at': '2023-03-08T12:46:24.472587', 'user': {'active': True, 'admin': True, 'groups': [], 'username': 'dev'}}]}
```

Reply:
This should use the preffered output set by the user, or text if not choosen.